docs/blueprints.rst

```diff
@@ -95,6 +95,43 @@ Variable Types
 Any native python type can be specified as the ``type`` for a variable.
 You can also use the following custom types:
 
+TroposphereType
+---------------
+
+The ``TroposphereType`` can be used to create one or more Troposphere
+types by directly passing in the value from the config to the specified
+Troposphere type.
+
+Example
+=======
+
+Below is an annotated example::
+
+  from stacker.blueprints.base import Blueprint
+  from stacker.blueprints.variables.types import TroposphereType
+  from troposphere import s3
+
+
+  class Bucket(Blueprint):
+
+      VARIABLES = {
+          "Buckets": {
+              # Specify that Buckets will be a list of s3.Bucket types.
+              This means the config should take a list of dictionaries
+              which will be converted into troposphere buckets.
+              "type": TroposphereType([s3.Bucket]),
```

Comment:
Is this, and a single type, the only applicable syntax for `TroposphereType`?  If so, probably worth documenting both.